packages/client/lib/execution/vmexecution.ts

```diff
@@ -278,7 +273,7 @@ export class VMExecution extends Execution {
       canonicalHead = await this.vm.blockchain.getLatestBlock()
     }
     this.running = false
-    return numExecuted as number
+    return numExecuted ?? 0
```

Comment:
Can't we default https://github.com/ethereumjs/ethereumjs-monorepo/blob/31391d176b8573939962046a9c8f45d8b9baf1c6/packages/client/lib/execution/vmexecution.ts#L137 to just set it diretly to 0? And also remove the `numExecuted === undefined` check in the loop?

Reply:
well not really because if 0 is returned (0 blocks executed) it should return and stop execution, not re-enter the while loop and there is nothing left to do.